tests/Tools/generate-stub-map

```diff
@@ -94,7 +94,8 @@ use const PHP_EOL;
         }
     }
 
-    $phpStormStubsDirectory = __DIR__ . '/../../';
+    $phpStormStubsDirectory = __DIR__ . '/../../src/';
+    $excludedStubDirNames = ['meta', 'couchbase_v2'];
```

Comment:
Not sure why these two need to be excluded - maybe it would be a good idea to add comments @isfedorov?
```php
// excluded because ...
$excludedStubDirNames = ['meta', 'couchbase_v2'];
```
or
```php
$excludedStubDirNames = [
    'meta',           // because this is meta
    'couchbase_v2',   // because some other reason
];
```
(second one could be nicer because of individual git history)